docs/start/getting-started/fragments/flutter/integrate.md

```diff
@@ -1,4 +1,4 @@
-In this section you’ll integrate Amplify DataStore with your app, and learn to use the generated data model to create, update, query, and delete Todo items by building an app. You can find the source code of the final Todo App [on our GitHub repository]().
+In this section you’ll integrate Amplify DataStore with your app, and learn to use the generated data model to create, update, query, and delete Todo items by building an app. You can find the source code of the final Todo App [on our GitHub repository](https://github.com/cshfang/amplify-flutter-datastore-starter/tree/main/amplified_todo).
```

Comment:
We will update these links when we have a more well-defined (and official) examples repo set up for amplify-flutter.